### PR TITLE
feat: feat/115/navbar-signed

### PR DESCRIPTION
**Pull request by Mwazowsky**
Fix navbar, signed in
image

sign out button
image

alert nya masih basic

### CODE REVIEW

File: src/lib/hooks/useNav.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    const handleLogout = () => {
+        const c = confirm("Are you sure you want to logout?");
+        if (c) {
+            Cookies.remove("accesstoken");
+            window.location.href = "/login";
+        }
+    };
+
+    const getInitials = (name: string) => {
```
Comment (bayuuat) [inline src/lib/hooks/useNav.ts]: aku belum set fullName kena breaking disini mas
image

**Review by bayuuat — CHANGES_REQUESTED** [review]

File: src/lib/hooks/useNav.ts (hunk 1 above)
Comment (Mwazowsky) [reply src/lib/hooks/useNav.ts]: Udah aku fix,
jadinya gini kalo belum set profile
image

**Review by bayuuat — APPROVED** [review]

**Comment by RutriAyut** [comment]
Makasih 👌

**Review by RutriAyut — APPROVED** [review]
Makasih 👍

**Pull request merged**